### ISSUE TITLE
Accessibility Issue for the Survey Creator Text Input property editor: Ensures every form element has a label

### ISSUE THREAD

**Issue by JaneSjs** [surveyjs/survey-library#5241]
Run [this example](https://plnkr.co/edit/TuJqo5r6X8My4OBc) and check it with the Accessibility Insights for Web checker. 
![image](https://user-images.githubusercontent.com/22372972/200816494-6903d9a9-fcf8-48e6-acea-608b225e2e0e.png)

Original issue: [T11204 - Accessibility issue: the input type doesn't have a label](https://surveyjs.answerdesk.io/internal/ticket/details/T11204)

[FastPass_20221109_NPSSurveyQuestionSur_T11204.zip](https://github.com/surveyjs/survey-creator/files/9970466/FastPass_20221109_NPSSurveyQuestionSur_T11204.zip)
